Header header
geometry_msgs/Pose GT_pose
geometry_msgs/Pose est_pose
geometry_msgs/Twist GT_twist
geometry_msgs/Twist est_twist
float32 RMSE_p
float32 RMSE_v